app/jobs/regular/post_update_topic_tracking_state.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Jobs
+  class PostUpdateTopicTrackingState < ::Jobs::Base
+
+    sidekiq_options queue: 'critical'
```

Comment:
We normally reserve `critical` for things like email.  Could we get away with the default queue here? Or does it lead to a poor user experience?

Reply:
My thinking was that since the code is currently being performed as part of the request, putting it in `critical` was just moving it one step down the priority list, if you catch my drift.

But I'm 100% happy to defer to your wisdom and put it on the default queue. I'll make the change.

Reply:
Yeah understood. I just think we should save that queue for things that are very important like "forgot password" emails and such. Let's try this and if it doesn't work we can consider a higher priority later.